modules/fedimint-ln-client/src/lib.rs

```diff
@@ -550,9 +571,12 @@ impl ClientModuleGen for LightningClientGen {
         Ok(LightningClientModule {
             cfg,
             notifier,
-            redeem_key: module_root_secret.child_key(ChildId(0)).to_secp_key(&secp),
+            outgoing_redeem_key: module_root_secret
+                .child_key(LIGHTNING_OUTGOING_CHILD_ID)
+                .to_secp_key(&secp),
```

Comment:
That's a gateway-only thing, right? Unrelated to this PR but should be documented in the struct.

Reply:
Which part is gateway only?

This key has been removed to fix the privacy regression. A new redeem key is generated each time an outgoing contract is created, so this isn't needed anymore.